src/doc/integrating_telio.md

```diff
@@ -0,0 +1,262 @@
+# Documentation For Integrating Telio Into Other Languages
```

Comment:
This weill need a bit more rework, so for now leave as empty, and let's introduce correct version from start